modules/fedimint-wallet-client/src/lib.rs

```diff
@@ -9,18 +9,23 @@
 pub mod api;
 
 pub mod client_db;
+/// Legacy, state-machine based peg-ins, replaced by `pegin_monitor`
+/// but retained for time being to ensure existing peg-ins complete.
 mod deposit;
```

Comment:
At some point in the future, we should be able to clean up and remove, but for now, it can just do what it was doing before.